### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by parijke**
Added sf version and php to requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Step-up Middleware Client Bundle
 [![Build Status](https://travis-ci.org/OpenConext/Stepup-Middleware-clientbundle.svg)](https://travis-ci.org/OpenConext/Stepup-Middleware-clientbundle) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenConext/Stepup-Middleware-clientbundle/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/OpenConext/Stepup-Middleware-clientbundle/?branch=develop) [![SensioLabsInsight](https://insight.sensiolabs.com/projects/3fef2be8-1351-4fbb-b6e8-00296c1d8d36/mini.png)](https://insight.sensiolabs.com/projects/3fef2be8-1351-4fbb-b6e8-00296c1d8d36)
 
-A Symfony2 bundle to consume the Step-up Middleware API. See [Stepup-Deploy](https://github.com/OpenConext/Stepup-Deploy) for an overview. 
+A Symfony6 bundle to consume the Step-up Middleware API. See [Stepup-Deploy](https://github.com/OpenConext/Stepup-Deploy) for an overview.
```
Comment (MKodde) [inline README.md:4]: Not sure why we had no whitespace before.. I suggest `Symfony 6`

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Step-up Middleware Client Bundle
 [![Build Status](https://travis-ci.org/OpenConext/Stepup-Middleware-clientbundle.svg)](https://travis-ci.org/OpenConext/Stepup-Middleware-clientbundle) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenConext/Stepup-Middleware-clientbundle/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/OpenConext/Stepup-Middleware-clientbundle/?branch=develop) [![SensioLabsInsight](https://insight.sensiolabs.com/projects/3fef2be8-1351-4fbb-b6e8-00296c1d8d36/mini.png)](https://insight.sensiolabs.com/projects/3fef2be8-1351-4fbb-b6e8-00296c1d8d36)
```
Comment (MKodde) [inline README.md:2]: You can replace the Travis badges with GithubActions counterparts!

:badger:

Comment (parijke) [reply README.md:2]: a sure... I think :-)

**Review by MKodde — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 # Step-up Middleware Client Bundle
-[![Build Status](https://travis-ci.org/OpenConext/Stepup-Middleware-clientbundle.svg)](https://travis-ci.org/OpenConext/Stepup-Middleware-clientbundle) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenConext/Stepup-Middleware-clientbundle/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/OpenConext/Stepup-Middleware-clientbundle/?branch=develop) [![SensioLabsInsight](https://insight.sensiolabs.com/projects/3fef2be8-1351-4fbb-b6e8-00296c1d8d36/mini.png)](https://insight.sensiolabs.com/projects/3fef2be8-1351-4fbb-b6e8-00296c1d8d36)
+[![Build status](https://github.com/OpenConext/Stepup-Middleware-clientbundle/actions/workflows/test-integration.yml/badge.svg)](https://github.com/SURFnet/sp-dashboard/actions/workflows/test-integration.yml)
```
[the remark this answers is not in the document]
Comment (parijke) [reply README.md:2]: Correct Sherlock

**Review by MKodde — APPROVED** [review]

**Pull request merged**